copy/copy.go

```diff
@@ -124,6 +128,15 @@ func Image(policyContext *signature.PolicyContext, destRef, srcRef types.ImageRe
 	if err != nil {
 		return errors.Wrapf(err, "Error initializing destination %s", transports.ImageName(destRef))
 	}
+	if options.Mounts != nil {
+		c, ok := dest.(docker.Mountable)
```

Comment:
No explicitly transport-specific code/imports in the generic `copy` interface, please.

In principle, this could just be another method in the `ImageDestination` interface.

But in this case, adding this as a specifically `docker`-specific option to `SystemContext`, completely invisible to `copy.Image`, would probably be more appropriate.

Reply:
Thanks for the review, SystemContext makes much more sense. Changed to that.